docs/verify/simulate-with-qiskit-primitives.ipynb

```diff
@@ -0,0 +1,584 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "235b9def-4c99-4511-87dc-fe972acd1249",
+   "metadata": {},
+   "source": [
+    "# Exact simulation with Qiskit primitives"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eda9656b-3403-40e3-a466-6007466adb93",
+   "metadata": {},
+   "source": [
+    "The reference primitives in Qiskit perform local statevector simulations. These simulations do not support\n",
+    "modeling device noise, but are useful for quickly prototyping algorithms before looking into more advanced simulation\n",
+    "techniques (using Qiskit Aer) or running on real devices (Qiskit Runtime primitives).\n",
```

Comment:
It would be nice to see links to relevant content for "more advanced simulation" and "running on real devices".